Kickstarter-iOS/Tests/ViewModels/UpdateViewModelTests.swift

```diff
@@ -64,7 +64,7 @@ final class UpdateViewModelTests: TestCase {
 
     withEnvironment(apiService: MockService(fetchUpdateResponse: prevUpdate)) {
 
-      let request = NSURLRequest(URL: optionalize(prevUpdateUrl)!)
```

Comment:
`optionalize` was just to help with swift 2.3 compatibility so can now get rid of it